.dotnet/src/Custom/Audio/AudioClient.cs

```diff
@@ -14,8 +14,8 @@ namespace OpenAI.Audio;
 /// <summary> The service client for OpenAI audio operations. </summary>
 [CodeGenClient("Audio")]
 [CodeGenSuppress("AudioClient", typeof(ClientPipeline), typeof(ApiKeyCredential), typeof(Uri))]
-[CodeGenSuppress("CreateSpeechAsync", typeof(SpeechGenerationOptions))]
-[CodeGenSuppress("CreateSpeech", typeof(SpeechGenerationOptions))]
+[CodeGenSuppress("CreateSpeechAsync", typeof(SpeechGenerationOptions), typeof(CancellationToken))]
+[CodeGenSuppress("CreateSpeech", typeof(SpeechGenerationOptions), typeof(CancellationToken))]
 [CodeGenSuppress("CreateTranscriptionAsync", typeof(AudioTranscriptionOptions))]
```

Comment:
nit: By any chance do you know why `CreateTranscription`, `CreateTranscriptionAsync`, `CreateTranslation` and `CreateTranslationAsync` don't need the `CancellationToken`? These methods are a little "special" because they can return different formats (i.e., they can return application/json, but also text/plain). I wonder if something about this is tripping the generator? Since these are "special" but also suppressed, it's not that important, so it's more curiosity than anything.

Reply:
I believe this is because these are multipart form data operations which will only get protocol methods. So we can actually remove these suppressions.